lib/services/auto_deploy.rb

```diff
@@ -33,7 +32,7 @@ def environment_names
   end
 
   def payload_ref
-    payload['ref'].split('/').last
+    payload['ref'].to_s[/refs\/heads\/(.*)/, 1]
```

Comment:
A test for this would make me feel more confident this change won't adversely impact existing installs.